<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibrateForm</class>
 <widget class="QWidget" name="CalibrateForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>480</width>
       <height>400</height>
      </size>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::West</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>4</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideLeft</enum>
     </property>
     <widget class="QWidget" name="tab_accelebrateCalibration">
      <attribute name="title">
       <string>加速度计校准</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_magnify">
      <attribute name="title">
       <string>罗盘校准</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_rc">
      <attribute name="title">
       <string>遥控器校准</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_motorTest">
      <attribute name="title">
       <string>电机测试</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_esc_cali_en">
      <attribute name="title">
       <string>电调校准</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
